x-pack/plugins/security_solution/public/resolver/store/data/reducer.ts

```diff
@@ -26,8 +28,21 @@ export const dataReducer: Reducer<DataState, ResolverAction> = (state = initialS
         },
       },
       resolverComponentInstanceID: action.payload.resolverComponentInstanceID,
+      locationSearch: action.payload.locationSearch,
+    };
+    const panelViewAndParameters = selectors.panelViewAndParameters(nextState);
+    return {
+      ...nextState,
+      // If the panel view or parameters have changed, the `nodeEventsInCategory` may no longer be relevant. In that case, remove them.
```

Comment:
Nice, I'll add the same for the `event_detail`

Reply:
@michaelolo24 No big deal if your reducer doesn't do the same. This will eventually hold, in memory, all data needed for the panel. If the user hit 'Load more' many times, this could be a lot of memory. That is one of my motivations for aggressively clearing it out. Also this was meant to simplify the logic for updating the collection of events when new data came back. If the data in reducer was guaranteed to be relevant, the code doesn't have to first verify it. Also the view won't accidentally show stale data.